### PR TITLE
MM-18761 Emoji picker hover effect should match the hover effect on post action menu items

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Change the opacity to match the hover effect on post menu actions

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-18761

After: Hovered on first emoji in the list

Screenshot 2019-10-15 at 3 40 11 PM
Screenshot 2019-10-15 at 3 39 51 PM

### CODE REVIEW

**Comment by sudheerDev** [comment]
@esethna For me it felt that the visual difference of hover seem too less now. Could be just me though. Can you check this on spinmint once if it feels right to you?

**Comment by esethna** [comment]
@sudheerDev can you help trigger the test server?

File: utils/utils.jsx [hunk 1]
```diff
@@ -792,7 +792,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .emoji-picker__category, .app__body .emoji-picker__category:focus, .app__body .emoji-picker__category:hover', 'color:' + changeOpacity(theme.centerChannelColor, 0.3));
         changeCss('.app__body .emoji-picker__category--selected, .app__body .emoji-picker__category--selected:focus, .app__body .emoji-picker__category--selected:hover', 'color:' + theme.centerChannelColor);
         changeCss('.app__body .emoji-picker__item-wrapper:hover', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
-        changeCss('.app__body .emoji-picker-items__container .emoji-picker__item.selected', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
+        changeCss('.app__body .emoji-picker-items__container .emoji-picker__item.selected', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.07));
```
Comment (esethna) [inline utils/utils.jsx]: Can we bump up the opacity to 30%?

File: utils/utils.jsx [hunk 2]
```diff
@@ -792,7 +792,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .emoji-picker__category, .app__body .emoji-picker__category:focus, .app__body .emoji-picker__category:hover', 'color:' + changeOpacity(theme.centerChannelColor, 0.3));
         changeCss('.app__body .emoji-picker__category--selected, .app__body .emoji-picker__category--selected:focus, .app__body .emoji-picker__category--selected:hover', 'color:' + theme.centerChannelColor);
         changeCss('.app__body .emoji-picker__item-wrapper:hover', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
-        changeCss('.app__body .emoji-picker-items__container .emoji-picker__item.selected', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
+        changeCss('.app__body .emoji-picker-items__container .emoji-picker__item.selected', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.30));
```
Comment (enahum) [inline utils/utils.jsx]:
```suggestion
        changeCss('.app__body .emoji-picker-items__container .emoji-picker__item.selected', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.3));
```

Comment (sudheerDev) [reply utils/utils.jsx]: @enahum Eric is trying out values. I am gonna wait for him to finish before committing this.

Comment (sudheerDev) [reply utils/utils.jsx]: Done. ES confirmed that he is good with 0.3

**Pull request merged**